.coveragerc

```diff
@@ -4,3 +4,4 @@ omit =
     tests/*
 source =
     gvmtools
+    scripts
```

Comment:
This did not work as expected. @bjoernricks how can I add the scripts directory to the coverage?

Reply:
I don't know. Did you take a look at the docs https://coverage.readthedocs.io/en/latest/ ?

Reply:
The problem is, that this PR is not running codecov ...